apps/ensadmin/src/hooks/useAvatarUrl.ts

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import { useNamespaceId } from "@/hooks/useNamespaceId";
+import { getNameAvatarUrl } from "@/lib/namespace-utils";
+import { Name } from "@ensnode/ensnode-sdk";
+import { useQuery } from "@tanstack/react-query";
+
+export interface UseAvatarUrlParameters {
+  name: Name;
+}
+
+/**
+ * Hook to get the avatar image URL for an ENS name using the ENS Metadata Service.
```

Comment:
```suggestion
 * Hook to get the URL for loading the avatar image for an ENS name through the ENS Metadata Service.
```

Important: We are _not_ loading the image URL from the ENS Metadata Service. That sounds like we send an API call to the ENS Metadata Service asking for the URL. That's not true.

Instead, we are simply formatting a string to a URL of the ENS Metadata Service. There are zero API calls here. We are just formatting a string.

All the language here needs to be reviewed and refined to clarify this.

Please study the logic inside `getNameAvatarUrl`.